cmd/systray/systray.go

```diff
@@ -558,7 +565,7 @@ type mullvadPeers struct {
 func (mp mullvadPeers) sortedCountries() []*mvCountry {
 	countries := slices.Collect(maps.Values(mp.countries))
 	slices.SortFunc(countries, func(a, b *mvCountry) int {
-		return cmp.Compare(a.name, b.name)
+		return cmp.Compare(strings.ToLower(a.name), strings.ToLower(b.name))
```

Comment:
this is a little painful to see allocs in a sort func

see https://en.wikipedia.org/wiki/Schwartzian_transform

I sent https://github.com/tailscale/tailscale/pull/14458

Reply:
fixed